pkg/utils/utils.go

```diff
@@ -1,9 +1,28 @@
 package utils
 
+import "fmt"
+
 // If implements ternary operator. if cond is true return v1, or return v2 instead.
 func If(cond bool, v1, v2 interface{}) interface{} {
 	if cond {
 		return v1
 	}
 	return v2
 }
+
+// FormatSize format image size to B/KB/MB/GB
+func FormatSize(size int64) string {
+	// we consider image size less than 1024 GB
```

Comment:
In code, what about numbers larger than 1024MB?
Maybe could be a little bit robust.